packages/@okta/vuepress-site/docs/guides/custom-widget-gen3/main/index.md

```diff
@@ -44,9 +48,21 @@ The third generation of the Sign-In Widget offers the same user experience as th
 
 See [Sign-In Widget, third generation](https://help.okta.com/okta_help.htm?type=oie&id=ext-compare-siw) to verify if the third generation is right for your org.
 
+### Palette generation and accessibility
+
+The third generation of the Sign-In Widget offers autogenerated palettes that preserve accessible color contrast ratios. Okta uses the [Odyssey design system](https://odyssey.okta.design/latest/foundations/accessibility/color-9O9a4XAU) to determine color contrast.
+
+Consider the following when adjusting the theming of your org:
+
+* If you don't modify your theming, your org uses the default Okta palette.
+
+* If you modify your theming using the **Theme** panel in the Admin Console, Okta generates an accessible palette. However, the generated palette might not map exactly to your brand's palette. See [Set a theme for your org](https://help.okta.com/okta_help.htm?type=oie&id=ext-branding-set-theme).
+
+* If you modify your theming using design tokens, you can create themes that exactly align with your brand's palette. For example, you can use the code editor to modify `PalettePrimaryMain` to have the exact hex code of your brand. However, if you use design tokens, you take on the responsibility of any broken accessibility ratios. See [Use the code editor](#use-the-code-editor).
```

Comment:
Maybe adding an internal anchor link to the design tokens section within this same page could make it even clearer how a reader can actually set something like a `PalettePrimaryMain` token?

Reply:
Good idea.